packages/kit/src/runtime/client/fetcher.js

```diff
@@ -90,6 +90,7 @@ export function initial_fetch(resource, opts) {
 
 	const script = document.querySelector(selector);
 	if (script?.textContent) {
+		script.remove(); // In case multiple script tags match the same selector
```

Comment:
What about this? Someone should benchmark which is faster (I would guess just reading is much faster)
```suggestion
	const [script, next_script] = document.querySelectorAll(selector);
	if (script?.textContent) {
		if (next_script) script.remove(); // In case multiple script tags match the same selector
```

Reply:
Scratch that, I perfed it an querySelectorAll is much slower